pkg/generators/extension_test.go

```diff
@@ -136,69 +141,58 @@ func TestExtensionErrors(t *testing.T) {
 
 	var tests = []struct {
 		comments []string
-		err      error
 	}{
 		{
 			// Missing extension value should be an error.
 			comments: []string{
 				"+k8s:openapi-gen=x-kubernetes-no-value",
 			},
-			err: errors.New("Invalid extension value: x-kubernetes-no-value"),
```

Comment:
I usually test stuff like this by having an e.g. "errorMessage string" and then check strings.Contains(err.Error(), errorMessage) if err is not nil. This lets me specify some portion of the string without being a full-on change detector, and the test continues to be useful if I use a different concrete type for the error in the future. In the previous take it was the "errors.New" part I objected to more than the string. Apologies for not being clear enough, I think it's best to do something halfway between the previous take and this take.